src/components/Carousels/TabCarousel.tsx

```diff
@@ -9,22 +9,22 @@ import { useCarousel } from '@hooks'
 import { TemplateCodeBlock } from '../TemplateCodeBlock'
 
 interface TabCarouselProps {
-    items: Template[]
+    items: TemplateProps[]
     currentItem?: ReactNode
     previousItem?: ReactNode
     currentItemIndex?: number
     autoAdvance?: boolean
 }
 
-interface Template {
+interface TemplateProps {
```

Comment:
Same question here on naming convention. To me, it doesn't make sense to have the suffix be `Props` if the interface isn't directly applied to a component, but is nested w/in. But maybe I'm misunderstanding our convention. Can you clarify, @bretthayes ?

Reply:
Yeah, I agree! I think we can forgo the Props suffix for interfaces that aren't directly applied to components. Let's go with that!